app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -23,10 +23,12 @@ export default async function fetchIndexPageProps(): Promise<IndexPageProps> {
 
   return {
     allDhbsVaccineDoseData: generateAllDhbsVaccineDoseData(
-      latestData.vaccinationsPerDhb,
-      yesterdayData.vaccinationsPerDhb
+      latestData.data.vaccinationsPerDhb,
+      yesterdayData.data.vaccinationsPerDhb
     ),
-    dataValidAsAtTimeUtc: DateTime.fromISO(latestData.dataValidAsAtNzTimeIso)
+    dataValidAsAtTimeUtc: DateTime.fromISO(
+      latestData.data.dataValidAsAtNzTimeIso
+    )
       .toUTC()
       .toISO(),
```

Comment:
Should this just be stored in UTC? Not part of this PR but something to consider 🙏

Reply:
yea agree, will do some other time in a diff PR